build.sbt

```diff
@@ -327,6 +327,16 @@ lazy val zincScripted = (project in internalPath / "zinc-scripted").
   ).
   configure(addSbtUtilScripted)
 
+lazy val crossTestBridges = {
+  Command.args("crossTestBridges", "<version>") { (state, args) =>
+    require(args.nonEmpty, "Missing Scala version argument.")
+    compilerBridgeScalaVersions.flatMap { (bridgeVersion: String) =>
+      s"wow $bridgeVersion" ::
+        s"${compilerBridge.id}/test" :: Nil
```

Comment:
If you're going to wow once (and you're also not wow-ing back), why don't you just plz?

`plz $bridgeVersion ${compilerBridge.id}/test`